src/components/pages/LocalIssues/LocalIssues.tsx

```diff
@@ -0,0 +1,31 @@
+import React, { useState } from 'react';
+
+import { AccordionUL, Bubble } from 'components';
+
+export default function LocalIssues() {
+  const [accordionFill, setAccordion] = useState([
```

Comment:
You can probably export this from a separate file and have the MDX import it. No need to create it in component state if you aren't doing anything with it